perceval/backends/git.py

```diff
@@ -154,7 +154,8 @@ def parse_git_log_from_file(filepath):
         :raises OSError: raised when an error occurs reading the
             given file
         """
-        with open(filepath, 'r', errors='surrogateescape') as f:
+        with open(filepath, 'r', errors='surrogateescape',
+                  newline = os.linesep) as f:
```

Comment:
Please follow PEP-8 rules about the use of whitespaces on keyword arguments.
```
Don't use spaces around the = sign when used to indicate a keyword argument
or a default parameter value. 
```
Change `newline = os.linesep` by `newline=os.linesep`